corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -216,8 +215,6 @@ def reprocess_serially(chunk, processor):
                             'change_ids': [c.id for c in changes_chunk]
                         })
                     self._record_batch_exception_in_datadog(processor)
-                    for change in set(changes_chunk) - set(retry_changes):
-                        self._record_change_success_in_datadog(change, processor)
```

Comment:
what's the reasoning behind removing this?

Reply:
this is in an exception block so `retry_changes` will have the value it had prior to the exception which is `set()` hence this would be recording all changes as success which is not true since there was an exception.

Reply:
thank you.